src/lib/y2storage/proposal/lvm_helper.rb

```diff
@@ -69,16 +61,9 @@ def initialize(planned_lvs, encryption_password: nil)
       #     partitions that should be added as PVs to the volume group
       # @return [Devicegraph]
       def create_volumes(original_graph, pv_partitions = [])
-        new_graph = original_graph.duplicate
-        return new_graph if planned_lvs.empty?
-
-        vg = reused_volume_group ? find_reused_vg(new_graph) : create_volume_group(new_graph)
-
-        assign_physical_volumes!(vg, pv_partitions, new_graph)
-        make_space!(vg)
-        create_logical_volumes!(vg)
-
-        new_graph
+        lvm_creator = LvmCreator.new(original_graph)
+        # FIXME: it won't be needed in the future
```

Comment:
I guess this FIXME does not apply to this line, but to the whole `LvmHelper#create_volumes` method.